### ISSUE TITLE
How to improve Quadro 2000M performance?

### ISSUE THREAD

**Issue by EstherMoellman** [CHEF-KOCH/nVidia-modded-Inf#20]
Hi @CHEF-KOCH ,

For my university studies and work I have a Lenovo' laptop, mainly because is a solid / resistant / powerful device, and in my environments I don't want nothing new, fancy or expensive. This Lenovo has a nVidia Quadro 2000M, I don't use it for games, but I use it a lot for mathematical and statistical graphs. Also, I use it a lot for video and media presentations... and it sucks, tons of pixelations and other ugly effects. I know... I know (LOL), this is not a great laptop, and this is not a great graphic-card... I can't expect miracles (LOL).

I did some research in the past, tried few nVidia tutorials, tweaked here and there... but really I saw zero improvements.
Here at your GitHub, I read you worked for nVidia, and I also read this your nVidia' repo. You're the expert I was looking for, so please I want to ask you:

- Can I improve my nVidia Quadro 2000M performance?

- Any recommended tweaks or configurations?

- Do you believe that by following your repo instructions, changing my INF-file, using the latest nVidia' driver etc, is this going to improve my nVidia Quadro 2000M?

You can see below my NVIDIA information. Hope you can help me.

Thank you in advance! : )

NVIDIA System Information report created on: 04/25/2020 06:23:01
System name: ABC

[Display]
Operating System:	Windows 10 Pro, 64-bit
DirectX version:	12.0 
GPU processor:		Quadro 2000M
Driver version:		377.83
Direct3D API version:	11.2
Direct3D feature level:	11_0
CUDA Cores:		192 
Core clock:		550 MHz 
Shader clock:		1100 MHz
Memory data rate:	1800 MHz
Memory interface:	128-bit 
Memory bandwidth:	28.80 GB/s
Total available graphics memory:	6118 MB
Dedicated video memory:	2048 MB DDR3
System video memory:	0 MB
Shared system memory:	4070 MB
Video BIOS version:	70.06.33.00.04
IRQ:			Not used
Bus:			PCI Express x16 Gen2
Device Id:		10DE 0DDA 21CF17AA
Part Number:		1319 

[Components]

nvui.dll		8.17.13.7783		NVIDIA User Experience Driver Component
nvxdsync.exe		8.17.13.7783		NVIDIA User Experience Driver Component
nvxdplcy.dll		8.17.13.7783		NVIDIA User Experience Driver Component
nvxdbat.dll		8.17.13.7783		NVIDIA User Experience Driver Component
nvxdapix.dll		8.17.13.7783		NVIDIA User Experience Driver Component
NVCPL.DLL		8.17.13.7783		NVIDIA User Experience Driver Component
nvCplUIR.dll		8.1.1070.0		NVIDIA Control Panel
nvCplUI.exe		8.1.1070.0		NVIDIA Control Panel
nvWSSR.dll		6.14.13.7783		NVIDIA Workstation Server
nvWSS.dll		6.14.13.7783		NVIDIA Workstation Server
nvViTvSR.dll		6.14.13.7783		NVIDIA Video Server
nvViTvS.dll		6.14.13.7783		NVIDIA Video Server
nvDispSR.dll		6.14.13.7783		NVIDIA Display Server
NVMCTRAY.DLL		8.17.13.7783		NVIDIA Media Center Library
nvDispS.dll		6.14.13.7783		NVIDIA Display Server
NVCUDA.DLL		6.14.13.7783		NVIDIA CUDA 8.0.0 driver
nvGameSR.dll		6.14.13.7783		NVIDIA 3D Settings Server
nvGameS.dll		6.14.13.7783		NVIDIA 3D Settings Server

**Comment by CHEF-KOCH** [comment]
Hey.

So before we start a discussion here is what you know about the GPU itself:
- The slow DDR3 memory in particular could be a bottleneck to the GPU
- OpenGL performance (one example) is better if you use Quadro series offers certified drivers compared to using a newer driver and make it compatible with your card
- The card has 55 Watt TDP
- The architecture is Fermi, the GPU name is GF106
- The product itself is end-of-life 
- Your BIOS is not uploaded to the [BIOS database](https://www.techpowerup.com/vgabios/?architecture=NVIDIA&manufacturer=&model=Quadro+2000M&interface=&memType=&memSize=&since=)
- You can e.g. "softmod" a GTS450 to a Quadro 2000 but Quadro to another card is (performance wise) not a good idea/possible.


> I did some research in the past, tried few nVidia tutorials, tweaked here and there... but really I saw zero improvements.

The problem is that Fermi supported has ended and driver wise there will no performance improvements at all because Nvidia removed the old code, so even if you would "somehow" manage to use a newer driver you would run into more problems.

> Any recommended tweaks or configurations?

No, you could try to mod the BIOS however, this is a) not easy b) can be more problematically because TDP/thermal throttling and overall decrease performance.

> Do you believe that by following your repo instructions, changing my INF-file, using the latest nVidia' driver etc, is this going to improve my nVidia Quadro 2000M?

No. 

**What I suggest**

You could use MSI Afterburner and try to overclock your card, make sure the temps are okay-ish so that you not run into BSOD's as well as thermal throttling.

**Comment by EstherMoellman** [comment]
@CHEF-KOCH ... totally understood, thank you! Honestly I was expecting a negative answer (LOL). Old laptop + old graphic card = no miracles (LOL).

Please, last question:
I'm not going to overclock my nVidia card. I also I'm not going to use the graphic card for games. And in this context of "no performance advantages" for my current Quadro 2000M... what technical advantages I have comparing my Quadro 200M with my Intel HD Graphics?

My graphic cards knowledge is almost zero. From my ignorance, I believe that my Intel HD Graphics will consume less energy, may be producing less hit, which overall may benefit CPU, battery and general performance. Am I wrong?
In my context, do I gain something really good, by using my Quadro 200M?
